lib/config/config-schema.json

```diff
@@ -0,0 +1,7 @@
+{
+  "Show": {
```

Comment:
Could you make the key lowercase? Atom's pattern [seems to be camel case with first letter lowercase](https://atom.io/docs/api/v1.6.0/Config)

Reply:
Fixed!